src/main/java/com/cloudant/client/org/lightcouch/Changes.java

```diff
@@ -215,7 +212,6 @@ private void setNextRow(Row nextRow) {
     }
 
     private void terminate() {
-        httpGet.abort();
```

Comment:
I haven't seen any issues with this.  Is it worth closing the `CouchDatabaseBase` object?